src/app/model/Rook.ts

```diff
@@ -0,0 +1,111 @@
+import { Board } from './Board';
+import { Piece } from './Piece';
+import { PieceMovement } from './PieceMovement';
+import { Side, Square } from './Types';
+
+export class Rook extends Piece implements PieceMovement {
+  readonly boardSize = 8;
+
+  constructor(id: string, side: Side) {
+    super(id, side);
+  }
+
+  possibleMoves(position: Square, board: Board): Square[] {
+    let movesToGo: Square[] = [];
```

Comment:
Ta zmienna już nam nie potrzebna :)

Reply:
Fakt, chciałem po prostu określić jej typ, żeby było wiadomo co to, ale to chyba faktycznie bez sensu skoro mam jako wynik funkcji : Square[]